### PR TITLE
Added a note about sonata_type_model requisite in 3.x branch

### PR DESCRIPTION
**Pull request by lelledaniele**

### CODE REVIEW

**Comment by lelledaniele** [comment]
Hi @OskarStark,

sorry I got a lot of conflicts to change the branch.
I created this new pull request.

Thanks

**Comment by jordisala1991** [comment]
https://github.com/sonata-project/SonataAdminBundle/pull/4623

Referencing old PR to not lose track of the conversation.

**Comment by jordisala1991** [comment]
Shouldn't we also change a little bit this text?

```
Since the ``image1`` field refers to a related entity we do not need to specify
 any options. Sonata will calculate that the linked class is of type ``Image`` and,
 by default, retrieve a list of all existing Images to display as choices in the
 selector.
```

Atleast to note that it won't retrieve by default al existing images... something like this:

```
Since the ``image1`` field refers to a related entity we do not need to specify
 any options. Sonata will calculate that the linked class is of type ``Image`` and,
 by default, use the ``ImageAdmin`` class to retrieve a list of all existing Images to display as choices in the selector.
```

WDYT?

**Comment by lelledaniele** [comment]
Hi @jordisala1991,

I just did a small change.
Now looks better for me.

Any other suggestions?

Thanks,
Lelle

File: Resources/doc/reference/form_types.rst [hunk 1]
```diff
@@ -40,18 +40,18 @@ All we need to do now is add a reference for this field in our ``PageAdmin`` cla
...
 
+Since the ``image1`` field refers to a related entity we do not need to specify
+any options. Sonata will calculate that the linked admin class is of type ``Image`` and,
+by default, retrieve a list of all existing Images to display as choices in the
+selector.
+
 .. tip::
 
-    You need to create ``ImageAdmin`` class in this case to use ``sonata_type_model`` type.
+    To create the linked ``ImageAdmin`` class in this case to use ``sonata_type_model`` type.
```
Comment (jordisala1991) [inline Resources/doc/reference/form_types.rst]: Why this change? now I don't understand this phrase? Wasn't better before?

Comment (lelledaniele) [inline Resources/doc/reference/form_types.rst]: Hi @jordisala1991,

I reverted this change.
My idea was using the word `linked` because in the paragraph above we use `linked admin class`.

Thanks,
Lelle

**Comment by jordisala1991** [comment]
Thanks @lelledaniele
